module/VuFind/src/VuFind/Db/Table/Tags.php

```diff
@@ -156,6 +158,7 @@ public function resourceSearch($q, $source = null, $sort = null,
             } else {
                 $select->where->equalTo('tags.tag', $q);
             }
+            $select->where->isNotNull('rt.resource_id');
```

Comment:
Should this get the standard "Discard tags assigned to a user list." comment too?